arrowFunctions.js

```diff
@@ -13,9 +15,32 @@ function modifyArray(nums) {
 function modifyArrayX(nums) {
   for (let i = 0; i < nums.length; i++) {
     if (nums[i] % 2 === 0)
-      nums[i] * 2;
+     nums[i] *= 2; 
     else
-      nums[i] * 3;
+     nums[i] *= 3;
   }
-  return nums;
-}
\ No newline at end of file
+   return nums;
```

Comment:
@Ayopal Check your code again on this. If you do it like this, it won't return an array.

Reply:
corrected.  Kindly advice on a snippet to use for auto indent